components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -1622,6 +1622,11 @@ public static String getOpenAPIDefinitionFilePath(String apiName, String apiVers
                 apiName + RegistryConstants.PATH_SEPARATOR + apiVersion + RegistryConstants.PATH_SEPARATOR;
     }
 
+    public static String getGraphqlDefinitionFilePath(String apiName, String apiVersion, String apiProvider) {
+        return APIConstants.API_ROOT_LOCATION + RegistryConstants.PATH_SEPARATOR + apiProvider + RegistryConstants.PATH_SEPARATOR +
```

Comment:
shall we use StringBuilder?

Reply:
Using a String concatenation makes the code shorter and simpler in this time. Once I add the StringBuilder to this, it gives the warning to change it to the concatenation